pages/linux/pacman-deptest.md

```diff
@@ -0,0 +1,20 @@
+# pacman --deptest
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --deptest --help`
+
+- List not installed packages:
```

Comment:
Is this equivalent to "list all packages on all repositories that are not locally installed" or "list non-explicitly installed packages" :thinking:

Reply:
No. This command is for AUR tools.
You pass a space-separated list of package and the output is which packages are not installed from that list.

Reply:
Aha! Thank you. Then how about this?
```suggestion
- Out of a list of packages, print which ones are not installed:
```